System/ContentFile.cpp

```diff
@@ -33,7 +33,6 @@ namespace RTE {
 
 	int ContentFile::Create(const char *filePath) {
 		SetDataPath(filePath);
-		SetFormattedReaderPosition(GetFormattedReaderPosition());
```

Comment:
No, they were technically required at the time due to the weirdness with how it loaded in a different way instead of relying on Serializable. It’s not needed anymore though.

Reply:
Thanks, I was curious if it was something like that.